lib/pg_ha_migrations/unsafe_statements.rb

```diff
@@ -50,6 +49,12 @@ def self.delegate_unsafe_method_to_migration_base_class(method_name)
   disable_or_delegate_default_method :remove_index, ":remove_index is NOT SAFE! Use safe_remove_concurrent_index instead for Postgres 9.6 databases; Explicitly call :unsafe_remove_index to proceed on Postgres 9.1"
   disable_or_delegate_default_method :add_foreign_key, ":add_foreign_key is NOT SAFE! Explicitly call :unsafe_add_foreign_key only if you have guidance from a migration reviewer in #service-app-db."
 
+  def unsafe_add_column(table, column, type, options = {})
+    safely_acquire_lock_for_table(table) do
```

Comment:
It's a little unfortunate that we now have an implicit dependency in this file on this method in `safe_statements`; but the only alternative I can see is calling `execute_ancestor_statement` in that file, which is not an improvement. Perhaps they could be moved to a separate helper module.